src/signalcorr.jl

```diff
@@ -420,6 +488,16 @@ function pacf!{T<:RealFP}(r::RealMatrix, X::Matrix{T}, lags::IntegerVector; meth
     return r
 end
 
+
+"""
+    pacf(X, lags; method=:regression)
+
+Compute the partial autocorrelation function (PACF) of `X` at `lags`. `method`
```

Comment:
Should mention that `X` can be a real-valued vector or matrix. Same for some functions above.